bigflow/build.py

```diff
@@ -33,16 +34,16 @@ def run_tests(build_dir: Path, test_package: Path):
 def export_docker_image_to_file(tag: str, target_dir: Path, version: str):
     image_target_path = target_dir / f'image-{version}.tar'
     print(f'Exporting the image to file: {image_target_path}' )
-    run_process(f"docker image save {get_docker_image_id(tag)} -o {resolve(image_target_path)}")
+    run_process(f"docker image save {get_docker_image_id(tag)} -o {image_target_path}")
 
 
 def build_docker_image(project_dir: Path, tag: str):
     print('Building a Docker image. It might take a while.')
-    run_process(f'docker build {resolve(project_dir)} --tag {tag}')
+    run_process(f'docker build {project_dir} --tag {tag}')
 
 
 def build_dags(
-        root_package,
+        root_package: Path,
```

Comment:
it would be nice if you would add typing in the whole module :)

Reply:
done